pkg/exporter/consul_exporter.go

```diff
@@ -61,6 +61,11 @@ var (
 		"Status of member in the cluster. 1=Alive, 2=Leaving, 3=Left, 4=Failed.",
 		[]string{"member"}, nil,
 	)
+	memberWanStatus = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "", "serf_wan_member_status"),
+		"Status of member in the wan cluster. 1=Alive, 2=Leaving, 3=Left, 4=Failed.",
```

Comment:
We typically do ENUMs like this as a "state set". It would look something like this:

```
consul_serf_wan_member_status{status="alive"} 1
consul_serf_wan_member_status{status="leaving"} 0
consul_serf_wan_member_status{status="left"} 0
consul_serf_wan_member_status{status="failed"} 0
```
The boolean status pattern has some easier aspects when combining metrics with joins.

Reply:
I did it like this becuase the memberStatus is like this and felt correct leaving both of them equal.
https://github.com/jorgemarey/consul_exporter/blob/f-serf-wan-member-status/pkg/exporter/consul_exporter.go#L59

Reply:
Ahh, I hadn't seen that. Probably something else to fix.

Reply:
Does that mean that I should fix it in this PR?

Reply:
No, let's fix it in a followup.